ceph/ceph_admin/bootstrap.py

```diff
@@ -41,8 +43,12 @@ def construct_registry(cls, registry: str, json_file: bool = False):
         constructed string of registry credentials ( Str )
     """
     # Todo: Retrieve credentials based on registry name
+    build_type = "ibm" if ibm_build else "rh"
```

Comment:
build_type = "rh" if not ibm_build else "ibm" - please add

Reply:
this is same condition, only change in string placements.